examples/elasticsearch-incremental/README.md

```diff
@@ -0,0 +1,243 @@
+cat <<EOF | kubectl apply -f -
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: quickstart
+spec:
+  version: 8.4.1
+  nodeSets:
+  - name: default
+    count: 2     
+    volumeClaimTemplates:
+    - metadata:
+        name: elasticsearch-data # Do not change this name unless you set up a volume mount for the data path.
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 100Gi  
+    config:
+      node.store.allow_mmap: false
+EOF
+```
+
+## Integrating with Kanister
+
+If you have deployed the elasticsearch cluster with other name 
+than `quickstart` and namespace other than `test-es1`, you need
+to modify the commands used below to use the correct name and namespace
+
+### Create Profile
+Create Profile CR if not created already
+
+```bash
+$ kanctl create profile s3compliant --access-key <aws-access-key-id> \
+	--secret-key <aws-secret-key> \
+	--bucket <s3-bucket-name> --region <region-name> \
+	--namespace test-es1
+```
+
+**NOTE:**
+
+The command will configure a location where artifacts 
+resulting from Kanister data operations such as backup 
+should go. This is stored as a profiles.cr.kanister.io 
+CustomResource (CR) which is then referenced in Kanister 
+ActionSets. Every ActionSet requires a Profile reference
+to complete the action. This CR (profiles.cr.kanister.io) 
+can be shared between Kanister-enabled application instances.
+
+
+### Create Blueprint
+Create Blueprint in the same namespace as the controller
+
+```bash
+$ kubectl create -f elasticsearch-incremental-blueprint.yaml -n kasten-io
+```
+
+Once Elasticsearch is running, you can populate it with some data. 
+
+Create a curl pod client : 
+```
+PASSWORD=$(kubectl get -n test-es1 secret 
+quickstart-es-elastic-user -o go-template='{{.data.elastic | base64decode}}')
+ES_URL="https://quickstart-es-http:9200"
+kubectl run -n test-es1 curl -it --restart=Never --rm --image ghcr.io/kanisterio/kanister-kubectl-1.18:0.81.0 --env="PASSWORD=$PASSWORD" --env="ES_URL=$ES_URL" --command bash 
+```
+
+In the curl pod shell
+```
+curl -u "elastic:$PASSWORD" -k "${ES_URL}"
+# List the index 
+curl  -k -u "elastic:$PASSWORD" -X GET "${ES_URL}/*?pretty"
+# Create an index 
+curl  -k -u "elastic:$PASSWORD" -X PUT "${ES_URL}/my-index-000001?pretty"
+#  add an index and a document 
+curl  -k -u "elastic:$PASSWORD" -X PUT "${ES_URL}/my-index-000002/_doc/1?timeout=5m&pretty" -H 'Content-Type: application/json' -d'
+{
+  "@timestamp": "2099-11-15T13:12:00",
+  "message": "GET /search HTTP/1.1 200 1070000",
+  "user": {
+    "id": "kimchy-2"
+  }
+}
+'
+# retreive document 
+curl -k -u "elastic:$PASSWORD" -X GET "${ES_URL}/my-index-000002/_doc/1?pretty"
+```
+
+
+## Protect the Application
+
+You can now take a backup of the elasticsearch data 
+using an ActionSet defining backup for this application. 
+Create an ActionSet in the same namespace as the controller.
```

Comment:
```suggestion
Create an ActionSet in the same namespace as the Kanister controller.
```

Since we are installing Elasticsearch operator as well, I think it's good that we talk about kanister controller explicitly.